server/src/main/java/io/spine/server/model/MethodSignature.java

```diff
@@ -78,24 +78,30 @@ protected MethodSignature(Class<? extends Annotation> annotation) {
     /**
      * Obtains the set of allowed access modifiers for the method.
      *
-     * <p>By default, obtains a set of single value {@link AccessModifier#PACKAGE_PRIVATE}. Most of
-     * the implementations should be fine with this behaviour. Override this method to change
+     * <p>Default implementation returns a set with a single value
+     * {@link AccessModifier#PACKAGE_PRIVATE}. Override this method to change
      * the allowed access modifiers.
      */
-    protected ImmutableSet<AccessModifier> allowedModifiers() {
+    protected ImmutableSet<AccessModifier> modifiers() {
         return ImmutableSet.of(PACKAGE_PRIVATE);
     }
 
     /**
      * Obtains the set of valid return types.
      */
-    protected abstract ImmutableSet<Class<?>> validReturnTypes();
+    protected abstract ImmutableSet<Class<?>> returnTypes();
 
     /**
-     * Obtains the set of allowed exceptions that method may declare to throw.
+     * Obtains the type of a {@code Throwable} which a method can declare.
+     *
+     * <p>A handler method may declare more than one {@code Throwable}, but they must be of
+     * the same super-type.
```

Comment:
I don't get this. Does this paragraph mean that we have to provide the broadest exception type?